datafusion/physical-expr/src/expressions/get_indexed_field.rs

```diff
@@ -42,61 +40,25 @@ use std::{any::Any, sync::Arc};
 pub enum GetFieldAccessExpr {
     /// Named field, For example `struct["name"]`
```

Comment:
🤔  maybe we should also convert this into a function call as well (as a follow on PR) to remove the entire `GetFieldAccessExpr` physical expr